guide/how-it-works/silent-payments.md

```diff
@@ -0,0 +1,360 @@
+   retina = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work@2x.png"
+   mobile = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work.png"
+   mobileRetina = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work@2x.png"
+   alt-text = "An image that illustrates how silent payments work in 4 steps."
+   caption = "."
+   width = 1024
+   height = 522
+   layout = "full-width"
+%}
+
+With this new model of transacting, BIP-352 introduces new primitives and concepts for users and developers. Below is a brief summary of them:
+
+| Current terminology        | With silent payments          | Benefit |
+|:-------------|:------------------|:------|
+| Single-use on-chain address           | Static address | Reusable  |
+| Extended private key (xprv) | Spend key   |   |
+| Extended public key (xpub)           | Scan key      | Addresses cannot be deterministically derived   |
+| **Address-based model**           | **Contact-based model** | **More intuitive and familiar**  |
+
+{% include image-gallery.html pages = page.images_create %}
+
+
+## Contacts & labels
+
+### Labels
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/silent-payments/adding-labels-to-static-address.png"
+   retina = "/assets/images/guide/how-it-works/silent-payments/adding-labels-to-static-address@2x.png"
+   modalImage = "/assets/images/guide/how-it-works/wallet-privacy/network-settings-big.png"
+   alt-text = "An application screen showing labels being added to a static address."
+   caption = "Adding labels to a static address before posting or sharing with others."
+   width = 250
+   height = 542
+   layout = "float-right-desktop -background -shadow"
+%}
+
+BIP-352 allows users to customize their silent payment addresses on a case-by-case basis by adding labels. Users may add labels to customise their static addresses before sharing or posting them. When a payment is received, these labels can be detected and used to identify the static address used to make the payment without it being obvious on-chain.
+
+</div>
+
+This scheme improves label-related features since any labels added to a static address get auto-applied to all derived on-chain addresses (detectable only by the receiver), eliminating the friction in manually adding labels to addresses or transactions. This not only makes payment tracking possible, but also useful during automatic or manual coin selection.
+
+For eg: a contractor Alice can add labels to her silent payment address while sharing them with a certain client, her social media and her website. When she receives payments from any of these sources, the respective label will be detected by the wallet application in the on-chain address to infer the source of the payment as this client.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/silent-payments/how-silent-payment-labels-work.png"
```

Comment:
This is a very complex diagram. Can you maybe remove some of the stuff that is already shown in the diagram above, and focus only on the label-specific information?

Reply:
Tried to address this in https://github.com/BitcoinDesign/Guide/pull/1109/commits/73d4e9128538a39f15ddf6f3b1972d5ddfdf6ad0 (or just see revised labels illustration in preview)

Reply:
Definitely a tricky one! It also helped me better understand how labels can be useful with silent payments. 

I took a stab at simplyfying it a bit. I tried to reduce the number of elements in the diagram. I also feel like it's easier to parse, if it's laid out vertically. Especially on mobile phones. I also added some annotations for added context directly in the diagram. People will probably click on the enlarged view, so annotations should be helpful. 

![image](https://github.com/user-attachments/assets/5c3d1b0a-1367-4061-a4c3-f737a4a75316)

Reply:
Illustrations have been updated incorporating themes from this excellent suggestion.